Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -735,4 +736,18 @@ protected boolean isConditionsDefaultValueEnabled() {
         }
         return Boolean.TRUE;
     }
+
+    public static String appendPrefixHomeCommunityID(final String homeCommunityId) {
+        return checkPrefixBeforeAppend(homeCommunityId, "urn:oid:");
```

Comment:
We already have this variable which defines under Use NhincConstants.HCID_PREFIX